### PR TITLE
fix: usd no pool & numberic mobile

### PR DESCRIPTION
**Pull request by athevinha**

### CODE REVIEW

File: src/Components/BuyPositionBox/index.tsx [hunk 1]
```diff
@@ -317,7 +317,8 @@ const Component = ({
           </div>
         </div>
         <Input
-          placeholder='0.0'
+          placeholder='0.0'
```
Comment (Zergity) [inline src/Components/BuyPositionBox/index.tsx:320]: unessesary change: trailing space

File: src/Components/Positions/index.tsx [hunk 2]
```diff
@@ -328,13 +328,13 @@ export const Positions = ({ setOutputTokenAddressToBuy, tokenOutMaturity }: { se
               <th>Position</th>
               <th className='no-wrap'>
                 Net Value
-                <Text
+                {wrapToNativeAddress(positions?.[0]?.pool?.TOKEN_R) ? <Text
```
Comment (Zergity) [inline src/Components/Positions/index.tsx:331]: only need to check `positions?.length > 0` here

File: src/Components/Positions/index.tsx [hunk 3]
```diff
@@ -328,13 +328,13 @@ export const Positions = ({ setOutputTokenAddressToBuy, tokenOutMaturity }: { se
...
                   className='text-link'
                   onClick={() => {
                     setValueInUsdStatus(valueInUsdStatus === VALUE_IN_USD_STATUS.USD ? VALUE_IN_USD_STATUS.TOKEN_R : VALUE_IN_USD_STATUS.USD)
                   }}
-                >{valueInUsdStatus === VALUE_IN_USD_STATUS.USD ? ` ⇄ ${tokens[wrapToNativeAddress(positions?.[0].pool.TOKEN_R)]?.symbol}` : ' ⇄ USD'}
-                </Text>
+                >{valueInUsdStatus === VALUE_IN_USD_STATUS.USD ? ` ⇄ ${tokens[wrapToNativeAddress(positions?.[0]?.pool?.TOKEN_R)]?.symbol}` : ' ⇄ USD'}
```
Comment (Zergity) [inline src/Components/Positions/index.tsx:336]: same comment above

File: src/Components/ui/Input/index.tsx [hunk 4]
```diff
@@ -6,9 +6,17 @@ type InputType = InputProps & {
...
+  isNumber?:boolean
 }
 
 export const Input = (props: InputType) => {
+  const isNumber = props?.isNumber || false;
+  const inputProps = isNumber ? {
+    type: 'number',
+    pattern: '[0-9]*',
+    inputMode: 'numeric',
+    placeholder: 'Nhập số'
```
Comment (Zergity) [inline src/Components/ui/Input/index.tsx:18]: English only please. And do we need the default `placeholder` here?

**Review by Zergity — CHANGES_REQUESTED** [review]

File: src/Components/Positions/index.tsx (hunk 3 above)
Comment (athevinha) [reply src/Components/Positions/index.tsx:336]: Comment above should be modified to read "positions.length > 0" in order to make the code functional.

File: src/Components/ui/Input/index.tsx (hunk 4 above)
Comment (athevinha) [reply src/Components/ui/Input/index.tsx]: I will remove `placeholder` code

File: src/Components/Positions/index.tsx (hunk 3 above)
Comment (Zergity) [reply src/Components/Positions/index.tsx:336]: why? what's wrong with `?.length`?

Comment (athevinha) [reply src/Components/Positions/index.tsx:336]: Sorry i miss `?.` in my comment, nothing wrong with use `?.length`

**Review by Zergity — APPROVED** [review]

**Pull request merged**